src/lib/features_at.js

```diff
@@ -24,5 +25,14 @@ module.exports = function(event, bbox, ctx) {
       return META_TYPES.indexOf(feature.properties.meta) !== -1;
     });
 
-  return sortFeatures(features);
+  var featureIds = new StringSet();
+  var uniqueFeatures = [];
+  features.forEach((feature) => {
+    const featureId = feature.id || feature.properties.id;
```

Comment:
That's where I was finding the ids in the features that Draw created. Fixing that to match the spec by putting the id directly on Feature could be another issue.